modules/orientdb/src/test/java/org/testcontainers/containers/OrientDBGremlinTest.java

```diff
@@ -0,0 +1,46 @@
+package org.testcontainers.containers;
+
+import org.apache.tinkerpop.gremlin.driver.Client;
+import org.apache.tinkerpop.gremlin.driver.Cluster;
+import org.apache.tinkerpop.gremlin.driver.ResultSet;
+import org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV3d0;
+import org.apache.tinkerpop.gremlin.orientdb.io.OrientIoRegistry;
+import org.apache.tinkerpop.gremlin.structure.io.gryo.GryoMapper;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * This test uses a preconfigured image of OrientDB 3.0..x with the gremlin server that expose the "demodb" on 8182 port.
+ *
+ * @author robfrank
+ */
+public class OrientDBGremlinTest {
+
+
+    @ClassRule
+    public static OrientDBContainer container = new OrientDBContainer("arcadeanalytics/orientdb:3.0.18-tp3")
```

Comment:
would be better to use the official image

Reply:
I removed this test